promises-demo-brightscript/components/MainScene.brs

```diff
@@ -0,0 +1,191 @@
+sub init()
+    initOnScreenImage()
+    m.top.observeFieldScoped("simpleResult", "onSimpleResultChange")
+    m.top.observeFieldScoped("chainResult", "onChainResultChange")
+    m.top.observeFieldScoped("parallelResult", "onParallelResultChange")
+
+    'Uncomment the example below to run...
+
+    'Simple single request
+    ' simpleExample("http://ip-api.com/json/")
+
+    'Chain requests
+    ' chainExample()
+
+    'Parallel requests
+    ' parallelExample()
+
+    'Simple single request with error response
+    ' simpleExample("http://invalid--url.com")
```

Comment:
I'd love to see a simple UI with a few buttons in it that trigger these various functions. That way devs don't have to uncomment the code to test out a specific flow.

Reply:
Can we merge what we have? ...will put this on my todo for the new year.